backend/v2/users/router.py

```diff
@@ -89,3 +100,76 @@ async def get_all_uqs_for_epoch_v1(
             for uq in all_uqs
         ]
     )
+
+
+@api.get("/{user_address}/antisybil-status")
+async def get_antisybil_status_for_user_v1(
+    session: GetSession,
+    uq_score_getter: GetUQScoreGetter,
+    # Request Parameters
+    user_address: Address,
+    response: Response,
+) -> AntisybilStatusResponseV1:
+    """
+    Returns user's antisybil status.
+    """
+
+    # If the user has no stamps, we return an unknown status
+    gp_stamps = await get_gp_stamps_by_address(session, user_address)
+    if gp_stamps is None:
+        response.status_code = 404
+        return AntisybilStatusResponseV1(
+            status="Unknown",
+            expires_at=None,
+            score=None,
+            is_on_time_out_list=None,
+        )
```

Comment:
maybe raise an exception instead of settings status_code to 404 manually? my feeling says it's more aligned with the standard

Reply:
please double check